topology_builder/pom.xml

```diff
@@ -33,18 +33,18 @@
     <dependency>
       <groupId>org.apache.storm</groupId>
       <artifactId>storm-core</artifactId>
-      <version>0.9.4</version>
```

Comment:
Thanks for the heads up. I'm in favor of bumping the version numbers, it's just that I'd like to have the opinion of another maintainer before merging the change.